mmv1/third_party/terraform/services/secretmanager/data_source_secret_manager_secret_version_test.go

```diff
@@ -74,6 +98,43 @@ func testAccCheckDatasourceSecretManagerSecretVersion(n, expected string) resour
 	}
 }
 
+func testAccCheckDatasourceSecretManagerSecretVersionSecretData(datasource, resource string) resource.TestCheckFunc {
```

Comment:
```suggestion
func testAccCheckSecretManagerSecretVersionSecretDataDatasourceMatchesResource(datasource, resource string) resource.TestCheckFunc {
```
(+ callers)

Reply:
I've updated the function name wherever required.